### ISSUE TITLE
mfsmount: sysctl(kern.proc.filedesc) error: ESRCH (No such process)

### ISSUE THREAD

**Issue by oszafraniec** [moosefs/moosefs#19]
We have a lot of those errors in our logs on FreeBSD servers. Remount doesn't help. It shows up all the time from time to time randomly.

I found that ESRCH means:
https://www.freebsd.org/cgi/man.cgi?query=errno&apropos=0&sektion=2&manpath=FreeBSD+10.0-RELEASE&arch=default&format=html

> 3 ESRCH No such process. No process could be found corresponding to that
>             specified by the given process ID.

Part of log files:

```
cat /var/log/messages | grep mfsmount
Sep 15 03:11:27 s2 mfsmount[46429]: sysctl(kern.proc.filedesc) error: ESRCH (No such process)
Sep 15 03:27:26 s2 mfsmount[46429]: sysctl(kern.proc.filedesc) error: ESRCH (No such process)
Sep 15 03:27:26 s2 mfsmount[46429]: sysctl(kern.proc.filedesc) error: ESRCH (No such process)
Sep 15 03:57:13 s2 mfsmount[46429]: sysctl(kern.proc.filedesc) error: ESRCH (No such process)
Sep 15 03:57:13 s2 mfsmount[46429]: sysctl(kern.proc.filedesc) error: ESRCH (No such process)
Sep 15 04:03:24 s2 mfsmount[46429]: sysctl(kern.proc.filedesc) error: ESRCH (No such process)
Sep 15 04:23:27 s2 mfsmount[46429]: sysctl(kern.proc.filedesc) error: ESRCH (No such process)
Sep 15 04:23:27 s2 mfsmount[46429]: sysctl(kern.proc.filedesc) error: ESRCH (No such process)
Sep 15 04:33:46 s2 mfsmount[46429]: sysctl(kern.proc.filedesc) error: ESRCH (No such process)
Sep 15 04:33:46 s2 mfsmount[46429]: sysctl(kern.proc.filedesc) error: ESRCH (No such process)
Sep 15 04:58:26 s2 mfsmount[46429]: sysctl(kern.proc.filedesc) error: ESRCH (No such process)
Sep 15 05:23:37 s2 mfsmount[46429]: sysctl(kern.proc.filedesc) error: ESRCH (No such process)
Sep 15 05:53:24 s2 mfsmount[46429]: sysctl(kern.proc.filedesc) error: ESRCH (No such process)
```

System info:

```
[root@s2 ~]# uname -a 
FreeBSD xxxx.xxx 10.0-RELEASE-p3 FreeBSD 10.0-RELEASE-p3 #0: Tue May 13 18:31:10 UTC 2014     root@amd64-builder.daemonology.net:/usr/obj/usr/src/sys/GENERIC  amd64
```

```
[root@s2 ~]# mfsmount -V
MFS version 3.0.81-1
FUSE library version: 2.9.5
mount_fusefs [fuse4bsd] version: 0.3.9-pre1
```

**Comment by xandrus** [comment]
Hi,
I would like to inform you, that we have found potential reason of this problem.
When new version will be available we let you know.

**Comment by xandrus** [comment]
I would like to inform you that we have published MooseFS 3.0.85 version.
We have made few bug fixes, and in particular, fixes fox FreeBSD MooseFS clients.

We strongly recommend to update MooseFS to latest version.
I'm looking forward to hearing from you.
